sale_order_product_recommendation/models/res_config_settings.py

```diff
@@ -4,14 +4,12 @@
 
 
 class ResConfigSettings(models.TransientModel):
-    _inherit = 'res.config.settings'
+    _inherit = "res.config.settings"
```

Comment:
This diff denotes that the pre-commit or the commit history hasn't been done correctly before, as this change should be done since 13.0.